app/store/models/Reaction.d.ts

```diff
@@ -13,4 +13,5 @@ export interface ReactionsGrouped {
   count: number;
   hasReacted: boolean;
   reactionId?: ID;
+  user?: string;
```

Comment:
Is this related to this PR? We haven't merged in the backend changes for this yet